### PR TITLE
feat: PRT staking page

### PR DESCRIPTION
**Pull request by 0xonramp**
Initial PRT staking layout.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "editor.codeActionsOnSave": {
     "source.organizeImports": "always"
-  }
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "typescript.preferences.importModuleSpecifier": "non-relative"
```
Comment (0xonramp) [inline .vscode/settings.json:7]: Favor full path imports for clarity, e.g. `@/app/prt-staking/constants`

File: src/app/prt-staking/constants.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { getIndexTokenData } from '@indexcoop/tokenlists'
+
+import { ProductRevenueToken } from '@/app/prt-staking/types'
+
+export const prts: ProductRevenueToken[] = [
+  {
+    tokenData: getIndexTokenData('hyETH')!,
```
Comment (0xonramp) [inline src/app/prt-staking/constants.ts:7]: Potentially could hardcode this in the future in tokenlists, e.g. `IndexEthereum.HYETH`

File: .vscode/settings.json (hunk 1 above)
Comment (janndriessen) [reply .vscode/settings.json:7]: The only thing I'd be wondering here is if then imports are not just one big pile - as organizeImports would all put it in the same block?

File: src/app/prt-staking/constants.ts (hunk 2 above)
Comment (janndriessen) [reply src/app/prt-staking/constants.ts:7]: Feel free to create a task in the tokenlists repo for discussion and with more infos on the approach.

File: src/app/prt-staking/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+export const metadata = {
+  title: 'PRT Staking',
+}
+
+export default function Layout({ children }: Props) {
+  return (
+    <div className="flex h-fit flex-col bg-[url('/presale-splash.jpg')] bg-cover">
+      <Providers>
+        {isLeverageSuiteEnabled() && <LeverageBanner />}
```
Comment (janndriessen) [inline src/app/prt-staking/layout.tsx:19]: Should this be `isStakingEnabled`?

Comment (janndriessen) [reply src/app/prt-staking/layout.tsx:19]: Ah no but don't you remove this in the next PR?

**Review by janndriessen — APPROVED** [review]
LGTM! 

Some small comments to check before merging.

File: src/app/prt-staking/layout.tsx (hunk 3 above)
Comment (0xonramp) [reply src/app/prt-staking/layout.tsx]: Yep - removed in the next PR, merged that in here

File: .vscode/settings.json (hunk 1 above)
Comment (0xonramp) [reply .vscode/settings.json:7]: Looks like it still keeps them separate :+1: 

https://github.com/IndexCoop/index-app/pull/1384/files#diff-8cc522524edba5381be22a9ad93761e10b46d58072beffcc7ee251b5ababf8c7R1-R5

File: src/app/prt-staking/constants.ts (hunk 2 above)
Comment (0xonramp) [reply src/app/prt-staking/constants.ts:7]: Added here: https://github.com/IndexCoop/tokenlists/issues/58

**Pull request merged**